packages/engine-server/src/doctor/service.ts

```diff
@@ -158,9 +159,15 @@ export class DoctorService {
     let doctorAction: (note: NoteProps) => Promise<any>;
     switch (action) {
       case DoctorActionsEnum.FIX_FRONTMATTER: {
-        console.log(
-          "the CLI currently doesn't support this action. please run this using the plugin"
-        );
+        const changed = await new BackfillService().updateNotes({
```

Comment:
This is not the right way to write doctor actions, you need to set `doctorAction` with an async function and place the code inside, like the other actions below. This also doesn't respect the scope selected by the doctor because of that.

Reply:
this is still a draft, just converted it to such